src/compiler/utilities.ts

```diff
@@ -6925,7 +6925,7 @@ namespace ts {
 
     export interface ObjectAllocator {
         getNodeConstructor(): new (kind: SyntaxKind, pos?: number, end?: number) => Node;
-        getTokenConstructor(): new <TKind extends SyntaxKind>(kind: TKind, pos?: number, end?: number) => Token<TKind>;
+        getTokenConstructor(): { new <TKind extends SyntaxKind>(kind: TKind, pos?: number, end?: number): Token<TKind> };

```

Comment:
NOTE: this just fixes coloration in the file. There seems to be a bug in the TypeScript syntax highlighting here.

Reply:
Filed as a bug here: https://github.com/Microsoft/TypeScript-TmLanguage/issues/687